python/indices_cff.py

```diff
@@ -107,8 +107,8 @@
 }
 muonRegions = {
     'signal': 'TopPairMuonPlusJetsSelection',
-    'controlRegion1': 'TopPairMuonPlusJetsQCDSelection1p5to3',
-    'controlRegion2': 'TopPairMuonPlusJetsQCDSelection3toInf',
+    'controlRegion1': 'TopPairMuonPlusJetsQCDSelection3toInf',
+    'controlRegion2': 'TopPairMuonPlusJetsQCDSelection1p5to3',
```

Comment:
The definition can be found here: [MuonUserData.cc#L185](https://github.com/BristolTopGroup/NTupleProduction/blob/master/plugins/userdata/MuonUserData.cc#L185)

I agree, this is easy to mix up, maybe we should rename it?

Anyway, 
`TopPairMuonPlusJetsQCDSelection3toInf == muon.userInt("isGoodNonIsoR1") == TopPairMuonPlusJetsQCDSelection3toInf` etc:
 - [muonSelections_cff.py#L13](https://github.com/BristolTopGroup/NTupleProduction/blob/master/python/muonSelections_cff.py#L13)
 - [miniAODToNTuple_cfg.py#L332](https://github.com/BristolTopGroup/NTupleProduction/blob/master/python/run/miniAODToNTuple_cfg.py#L332)